src/components/FormComponents/ServicecardInfo.js

```diff
@@ -0,0 +1,64 @@
+import styled from 'styled-components'
+import Button from '../Button/Button'
+
+export default function ServicecardInfo({
+  counter,
+  currentCosts,
+  setUsedCosts,
+  onAddingNewCosts,
+  index,
+}) {
+  return (
+    <>
+      <ServiceInfoForm
+        data-testid="setCostsForm"
+        onSubmit={event => handleSettingNewCosts(event)}
+      >
+        <div>Zeit: {counter} Stunden</div>
+        <div>
+          <label>
+            Stundensatz:
+            <NoSpinnersInput
```

Comment:
This name is not really descriptive, how about:
```suggestion
            <CostInput
```